doc/use_cli_effectively.md

```diff
@@ -131,6 +131,34 @@ This becomes an issue because when the command shell (bash, zsh, Windows Command
   1. If only `create/update` are involved, consider using `az group deployment create`. Leverage [Azure Quickstart Templates](https://github.com/Azure/azure-quickstart-templates) for working examples.
   2. Check out the Rest API reference for the request payload, URL and API version. As an example, check out the community's comments on [how to create AppInsights](https://github.com/Azure/azure-cli/issues/5543).
 
+## Using Storage Commands
+1. Useful environment variables for storage related commands:
+- `AZURE_STORAGE_ACCOUNT`: Storage account name. Must be used in conjunction with either storage account key or a SAS token. If neither are present, the command will try
+                           to query the storage account key using the authenticated Azure account. If a large number of storage commands are executed the API quota may be hit.
```

Comment:
Can you be more specific here?  Define "large number" and "may be hit" with some clearer verbiage.